### PR TITLE
Feature: Added Auth With pswd hash

### PR DESCRIPTION
**Pull request by ChaudharyRaman**

### CODE REVIEW

File: src/auth/auth.service.ts [hunk 1]
```diff
@@ -1,32 +1,73 @@
...
-      return Promise.resolve(null);
+    } catch (error) {
+      return Promise.resolve({
+        message: 'error',
+        error: error,
+      });
     }
   }
 
-  async createUser(data: User): Promise<any> {
```
Comment (varun7singh) [inline src/auth/auth.service.ts:22]: let this be for now

Comment (varun7singh) [reply src/auth/auth.service.ts:22]: let the createUser endpoint be there

**Review by varun7singh — CHANGES_REQUESTED** [review]

File: src/auth/auth.service.ts (hunk 1 above)
Comment (ChaudharyRaman) [reply src/auth/auth.service.ts:22]: I changed the end point to more fitting name "register"

**Comment by varun7singh** [comment]
@ChaudharyRaman  remove package-lock.json

**Pull request merged**